md/widgets.md

```diff
@@ -9,6 +9,36 @@ A page consists of widgets that are arranged in rows, and optionally, containers
 
 Each widget in the palette has a description. To show a widget description, just roll over the widget in the palette.
 
+## Widget Actions
+
+:::info
+ Not available for containers.
+:::
+
+Close to the widget name in the properties panel, several options are available in widgets actions button.
+
+* Use **view** menu option to view the code of standard widget. Use standard widgets as a template for a custom widget, click on **Save as**, find a name for the custom widget, and make it your own.
```

Comment:
"Use **view** menu option to view the code of the standard widget in the widget editor. 
From there, you can clone this widget (to modify or extend it) by clicking on **Save as**. This will create a custom widget that you can modify as needed."